builder/lxc/step_lxc_create.go

```diff
@@ -63,6 +63,22 @@ func (s *stepLxcCreate) createFromRootFs(containerName string, config RootFsConf
 	return rootfs, err
 }
 
+func (s *stepLxcCreate) loadSidedisk(containerName, archivePath string, destDir string) (error) {
+	lxcDir := "/var/lib/lxc"
```

Comment:
I'm seeing this definition in 3 differents methods, maybe could be a var of *stepLxcCreate*